### DISCUSSION TITLE
Display a message with multiple lines

### DISCUSSION THREAD

**Discussion by mikeliu888** [Blazored/Toast#162] [Q&A] [answered]
I am trying to display a toast with multiple lines of message. I tried \r\n, &lt;br&gt;, &lt;br /&gt;, &lt;p&gt;. None of them worked. Anyone has an idea how to do it?

**Comment by chrissainty** [comment, marked as answer]
You'd need to use the custom component option to achieve this. Details of how to do that are in the readme.
